SpanishWords.Web/Controllers/TranslateApiController.cs

```diff
@@ -24,57 +22,76 @@ public TranslateApiController(IWordRepository wordRepository, ILogger<WordReposi
 
         //localhost:7057/api/TranslateApi/translate?word=a
         [HttpGet("translate")] 
-        public List<DTOWord> GetEnglishToSpanishTranslation(string word)
+        public APIWordsDTO GetEnglishToSpanishTranslation(string word)
         {
-            if (CheckInput(word) == false)
+            APIWordsDTO response = new APIWordsDTO();
+            if (CheckStringInput(word) == false)
             {
                 _logger.LogError(ExceptionHelper.METHOD_EMPTY_PARAMETER);
-                return new List<DTOWord>();
+                response.APIResponse.SetAllFields(ApiHelper.API_INPUT_PROCESS_ERROR, success: false, error: true);
+                response.Words = new List<WordDTO>();
```

Comment:
Nie lepiej przenieść to obiektu jako wartość domyślną zamiast tworzyć tutaj na sztywno?

Reply:
Done.